src/debugger/Debugger.cpp

```diff
@@ -166,20 +166,22 @@ bool Debugger::doEval(ExecutionState* state, ByteCodeBlock* byteCodeBlock, uint8
     uint8_t type = (uint8_t)(buffer[0] + 1);
     uint32_t size;
     char* data;
+    uint32_t index;
 
     memcpy(&size, buffer + 1, sizeof(uint32_t));
+    memcpy(&index, buffer + 1 + sizeof(uint32_t), sizeof(uint32_t));
```

Comment:
I think there is a minimum message size check somewhere, that should be updated as well.